web_widget_dropdown_dynamic_example/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import models
```

Comment:
Line at the end

Reply:
:) Don't we have a linter for that?

Reply:
That's the weird thing... It's not detected right now

Reply:
Yep, 100%, on other modules as well